### DISCUSSION TITLE
how to get rid of .bin files on Playstation gamelist

### DISCUSSION THREAD

**Discussion by norbertroche** [Rparadise-Team/Koriki#5] [Q&A]
Hi, i got a question: in the Playstation gamelist appears all bin files. how do I dissapear them? I tried in the scrips for consoles eliminating the .bin and other unnecesary extension but nothing happened? any tip? 
BTW: Awesome project!! Keep the good work!!

**Comment by eduardofilo** [comment]
Do you mean editing this file?

https://github.com/Rparadise-Team/Koriki/blob/main/base/.simplemenu/section_groups/consoles.ini

It should work removing .bin at the end, i. e. leaving PLAYSTATION config as this:

```
[PLAYSTATION]
execs = /mnt/SDCARD/.simplemenu/launchers/pcsx_rearmed_libretro,/mnt/SDCARD/.simplemenu/launchers/pcsx_rearmed_libretro_old,/mnt/SDCARD/.simplemenu/launchers/pcsx_rearmed_libretro_shaun
romDirs = /mnt/SDCARD/Roms/PS/
romExts = .pbp,.chd,.zip,.cue,.img,.iso,.m3u
```

**Reply by norbertroche** [reply]
Done!!! Thanks a lot!!! Officially my new OS for the Miyoo Mini!!! Keep the amazing work!!!!
